interop-testing/src/main/java/io/grpc/testing/integration/XdsTestClient.java

```diff
@@ -167,8 +169,10 @@ private void parseArgs(String[] args) {
               + c.qps
               + "\n  --rpc=STR              Types of RPCs to make, ',' separated string. RPCs can "
               + "be EmptyCall or UnaryCall. Default: UnaryCall"
+              + "\n[deprecated] Use XdsUpdateClientConfigureService"
```

Comment:
I don't understand this. How to use `XdsUpdateClientConfigureService` to change the value, when `runQps()` starts immediately?

Reply:
By default, the test client just starts up and sends unary RPCs. That's the warmup phase, the test case won't begin until the test driver has checked if there is at least one RPCs succeed on each backend. This is to ensure the backend service configuration has been picked up by TD. We do not care what type of RPCs used to do the warm up, default unary is completely fine. After the warmup phase, we can change call behaviors to whatever needed for achieving the test goal.

Reply:
But `rpcTypes` is only read once in `runQps()`, how can `rpcTypes` be used after updated by the driver?

Reply:
Oops, this is screwed up. Currently a `PeriodicRpc` task is created per RpcType when the test client starts. It should be changed to run a single `PeriodicRpc` and each time it runs, it sends out one RPC per RpcType.

Reply:
Should be fixed now.